api/handlers/exporter_test.go

```diff
@@ -17,9 +18,14 @@ import (
 	"github.com/stretchr/testify/require"
 	"go.uber.org/zap"
 
+	"github.com/ssvlabs/ssv/api"
+	model "github.com/ssvlabs/ssv/exporter/v2"
```

Comment:
What do you think about `exportertypes` or `exporterv2`? IMO `model` isn't easy to understand what it is when it's referenced in the code